server/services/voip/service.ts

```diff
@@ -17,6 +18,43 @@ export class VoipService extends ServiceClass implements IVoipService {
 		this.VoipServerConfiguration = new VoipServerConfigurationRaw(db.collection('rocketchat_voip_server_configuration'));
 	}
 
+	async addServerConfigData(config: Omit<IVoipServerConfig, '_id' | '_updatedAt'>): Promise<boolean> {
+		const { type } = config;
+
+		const existingConfig = await this.getServerConfigData(type);
+		if (existingConfig) {
+			throw new Error(`Error! There already exists an active record of type ${ type }`);
+		}
+
+		await this.VoipServerConfiguration.insertOne(config);
```

Comment:
Return this right away, since the return value is not sent as an API response
```suggestion
		return !!await this.VoipServerConfiguration.insertOne(config);
```

(not sure if eslint will like the !!await 🙈 )